src/v0/destinations/zendesk/utils.js

```diff
@@ -0,0 +1,17 @@
+const {
+  ZENDESK_MARKETPLACE_NAME,
+  ZENDESK_MARKETPLACE_ORG_ID,
+  ZENDESK_MARKETPLACE_APP_ID,
+} = require("./config");
+
+const getDefaultHeaders = () => {
```

Comment:
need a function for this?
Looks constant to me

Reply:
Agreed. Refactored it.